tests/unit/steps/task/test_generate_embeddings.py

```diff
@@ -23,7 +23,7 @@
 @pytest.fixture(scope="module")
 def transformers_llm() -> Generator[TransformersLLM, None, None]:
     llm = TransformersLLM(
-        model="sentence-transformers/all-MiniLM-L6-v2",
+        model="TaylorAI/bge-micro-v2",
```

Comment:
Maybe we could just create our own tiny random model for this too, see https://huggingface.co/distilabel-internal-testing/tiny-random-mistral

Reply:
I created an issue for this